src/Administration/Resources/app/administration/src/module/sw-settings/component/sw-settings-item/index.js

```diff
@@ -25,12 +25,6 @@ export default {
             type: Boolean,
             default: false,
         },
-        backgroundEnabled: {
```

Comment:
can i do this or is it a breaking change? the prop is useless as the icons in the new UI don't have background anymore

If we consider this a break I would deprecate the prop for 6.8, but already remove the usage in the template 

@jleifeld @seggewiss

Reply:
Can be removed. It doesn't make sense to contain a property which doesn't have any effect. Best way would be if it would be already removed or deprecated with the settings page change